Make clean failed on ubuntu 2204

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->

**What happened**:

```
# make clean
git checkout bpf/kmesh/bpf2go/bpf2go.go config/kmesh_marcos_def.h mk/api-v2-c.pc mk/bpf.pc bpf/kmesh/include/config.h
error: pathspec 'bpf/kmesh/include/config.h' did not match any file(s) known to git
make: *** [Makefile:157: clean] Error 1
```

This is introduced in #264

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kmesh version:
- Others:

--- a/Makefile
+++ b/Makefile
@@ -71,7 +71,7 @@ TMP_FILES := bpf/kmesh/bpf2go/bpf2go.go \
 	config/kmesh_marcos_def.h \
 	mk/api-v2-c.pc \
 	mk/bpf.pc \
-	bpf/kmesh/include/config.h
+	bpf/kmesh/ads/include/config.h
 
 .PHONY: all install uninstall clean build docker
 

